lib/providers/source_provider.dart

```diff
@@ -342,14 +343,15 @@ class SourceProvider {
     Codeberg(),
     FDroid(),
     IzzyOnDroid(),
-    Mullvad(),
-    Signal(),
+    FDroidRepo(),
     SourceForge(),
     APKMirror(),
-    FDroidRepo(),
-    SteamMobile(),
-    TelegramApp(),
+    Mullvad(),
+    Signal(),
     VLC(),
+    // WhatsApp(), // As of 2023-03-20 this is unusable as the version on the webpage is months out of date
```

Comment:
It seems that the APK on their website is now up to date. The version on the website (2.23.11.78) seems to be the same as the latest version on APKmirror.  It is even newer than the version on Google Play (for my device anyway - 2.23.11.77)